src/Mono.Android/java/mono/android/GCUserPeer.java

```diff
@@ -0,0 +1,19 @@
+package mono.android;
+
+class GCUserPeer {
```

Comment:
It could, it doesn't need to. We load monodroidAddReference off of the class directly using GetMethodID. In fact, monodroid-glue doesn't use IGCUserPeer at all. We fetch a gref to it.

Would you *prefer* I have `GCUserPeer` implement `IGCUserPeer`?